src/app/views/common/monaco/Monaco.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import MonacoEditor from 'react-monaco-editor';
+import MonacoEditor, { ChangeHandler } from 'react-monaco-editor';
 
 import './monaco.scss';
 
 interface IMonaco {
   body: object|undefined;
+  onChange?: ChangeHandler | undefined;
```

Comment:
I don't think this should be optional, no?

Reply:
It is optional because there are instances where we only want to hydrate the editor and don't expect a user to modify the content.

Reply:
gotcha